Client/RecommendationRequestClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Client;
+
+interface RecommendationRequestClient
+{
+    /**
+     * @param int $userId
+     * @param int $scenarioId
+     * @param int $locationId
+     * @param int $limit
+     * @return string|null JSON response
```

Comment:
Function should return raw YC response which is in json format.
There is a parameter `$format` which allows you to choose what output format do you want (from YC). BTW, perhaps this can be a good idea to place `$format` as an function input argument.